<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout_3" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Parametros Ventana</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QTextEdit" name="textEdit" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Modelo de Velocidad</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QTextEdit" name="textEdit_2" >
          <property name="textInteractionFlags" >
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences" >
    <property name="title" >
     <string>&amp;Preferencias</string>
    </property>
    <addaction name="actionSizeSettings" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuVelocityModel" >
    <property name="title" >
     <string>&amp;Modelo de Velocidad</string>
    </property>
    <addaction name="actionCreateModel" />
    <addaction name="actionLoadModel" />
    <addaction name="actionEditModel" />
    <addaction name="actionModelParams" />
    <addaction name="actionPreview" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuSimulation" >
    <property name="title" >
     <string>&amp;Simulacion</string>
    </property>
    <addaction name="actionSimParams" />
    <addaction name="actionRun" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuPreferences" />
   <addaction name="menuVelocityModel" />
   <addaction name="menuSimulation" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionSizeSettings" >
   <property name="text" >
    <string>&amp;Tama√±o de Ventanas</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionModelParams" >
   <property name="text" >
    <string>&amp;Parametros del Modelo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionPreview" >
   <property name="text" >
    <string>&amp;Vista Previa</string>
   </property>
  </action>
  <action name="actionEditModel" >
   <property name="text" >
    <string>&amp;Editar Modelo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSimParams" >
   <property name="text" >
    <string>&amp;Parametros de Simulacion</string>
   </property>
  </action>
  <action name="actionRun" >
   <property name="text" >
    <string>&amp;Ver Simulacion</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>Nuevo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>G&amp;uardar como</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Guardar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;Sobre Viper</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Abrir</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCreateModel" >
   <property name="text" >
    <string>Crear Modelo</string>
   </property>
  </action>
  <action name="actionLoadModel" >
   <property name="text" >
    <string>Cargar Modelo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>168</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
